examples/index.json

```diff
@@ -2687,6 +2687,14 @@
         "verifyOutput": false,
         "disablePlayground": true,
         "isLearnByExample": false
+      },
+      {
+        "name": "Query Complexity",
+        "url": "graphql-service-query-complexity",
+        "verifyBuild": true,
+        "verifyOutput": false,
+        "disablePlayground": true,
+        "isLearnByExample": false
```

Comment:
Isn't it better to add this under GraphQL advance features?

Reply:
I was also thinking the same and thought of going with the security. But I agree, this is better suited for the advanced features.